workflows/edit_eml/series_identifier.Rmd

```diff
@@ -0,0 +1,38 @@
+# Series Identifier (SID)
+A series identifier is a system metadata field that represents a persistent identifier for a data package.  These are useful for maintaining a single identifier when a data package is expected to receive updates (usually additional data) in the future.  
```

Comment:
I would alter the first sentence to be:
"A series identifier is a system metadata field that represents a single identifier across _multiple versions_ of a data package, a feature often requested by `submitter`s."